boards/arm/contextualelectronics_abc/contextualelectronics_abc.dts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2020 Analog Life LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "nRF52840 BLE Cell";
+	compatible = "nordic,nrf52840-ble-cell";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	status = "okay";
+	current-speed = <115200>;
+
+	tx-pin = <7>;
+	rx-pin = <6>;
+};
+
+arduino_serial: &uart1 {
+	status = "okay";
+	current-speed = <115200>;
+
+	/* UART Pin info. */
+	rx-pin  = <20>;
+	tx-pin  = <24>;
+	rts-pin = <17>;
+	cts-pin = <16>;
+
+	/* Use hardware flow control. */
+	hw-flow-control;
+
+	/* BG9x description. */
+	quectel_bg9x {
+		compatible = "quectel,bg9x";
+		label      = "quectel-bg9x";
+		status     = "okay";
+
+		mdm-power-gpios = <&gpio1 5  0>;
+		mdm-reset-gpios = <&gpio1 6  0>;
+		mdm-dtr-gpios   = <&gpio0 22 0>;
+	};
+};
+
+arduino_i2c: &i2c0 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	sda-pin = <27>;
+	scl-pin = <26>;
+
+	/* This I2C device is interfaced with the "BQ25895R" charger. */
```

Comment:
If that's the case, why isn't there a subnode for it here? Even if there's no driver, it seems convenient to have a binding and a node, to be able to get its address etc.

Reply:
I'm working on upstreaming the bq2589x driver for zephyr (https://github.com/bwasim/zephyr/tree/bq25895-zephyr), And the bindings / driver haven't been finalized yet. I will remove this comment for now and add the device tree node whenever I upstream the driver.